### PR TITLE
Fixes some

### PR DESCRIPTION
**Pull request by mattn**
* Use `path/filepath` instead of `path`
* Call `strings.TrimSpace` before checking empty since Windows return `\r\n`
* Use $USERPROFILE instead $HOME on Windows

### CODE REVIEW

**Comment by grijul** [comment]
Thank you for your contribution. Let me review the changes (file changes looks good) and I'll merge.

File: internal/args/args.go [hunk 1]
```diff
@@ -515,6 +513,7 @@ func printZAuthOtpTable() error {
 	tbl := table.New("ISSUER", "IDENTIFIER", "TYPE", "OTP", "REMAINING")
 
 	tbl.WithPadding(5)
+	tbl.WithWidthFunc(runewidth.StringWidth)
```
Comment (grijul) [inline internal/args/args.go:516]: Why are we using WithWidthFunc() here? Is this trying to solve any particular usecase?
As per documentation: _WithWidthFunc sets the function used to calculate the width of the string in a column. By default, the number of utf8 runes in the string is used_

Seems like default should suffice .. Am I missing something?

**Review by grijul — CHANGES_REQUESTED** [review]
Thank you for your time and effort. Overall looks good. Once we resolve the comments, this is good to merge.

File: internal/args/args.go (hunk 1 above)
Comment (mattn) [reply internal/args/args.go:516]: String Width is not same as Rune Count. For example `あ` have 2 cells in terminal but count of runes is 1. `runewidth.StringWidth` return 2.

Comment (grijul) [reply internal/args/args.go:516]: Got it! Thanks!

**Review by grijul — APPROVED** [review]

**Pull request merged**